netam/sequences.py

```diff
@@ -1,21 +1,38 @@
 """Code for handling sequences and sequence files."""
 
 import itertools
+import re
 
 import torch
 import numpy as np
 
 from Bio import SeqIO
 from Bio.Seq import Seq
 
+BASES = ("A", "C", "G", "T")
 AA_STR_SORTED = "ACDEFGHIKLMNPQRSTVWY"
-NT_STR_SORTED = "ACGT"
+# Add additional tokens to this string:
+RESERVED_TOKENS = "^"
+
+
+NT_STR_SORTED = "".join(BASES)
+BASES_AND_N_TO_INDEX = {base: idx for idx, base in enumerate(NT_STR_SORTED + "N")}
+# ambiguous must remain last
```

Comment:
I believe this but would like to know why.

Reply:
Added more comment